scripts/bump.sh

```diff
@@ -59,20 +59,37 @@ echo "::set-output name=version::$NEXT_EXTENSION_VERSION"
 jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
     .prisma.version = \"$SHA\" | \
     .dependencies[\"@prisma/get-platform\"] = \"$NPM_VERSION\"" \
-    ./../language-server/package.json > ./../language-server/package.json.bk
+    ./packages/language-server/package.json > ./packages/language-server/package.json.bk
 
-mv ./../language-server/package.json.bk ./../language-server/package.json
-mv ./package.json.bk ./package.json
+jq ".dependencies[\"@prisma/language-server\"] = \"$NEXT_EXTENSION_VERSION\"" \
+    ./packages/vscode/package.json > ./packages/vscode/package.json.bk
```

Comment:
This line is incorrect, it happens before last changes to vscode `package.json` are moved and overwrites them. 

Fixing this to create a patch.